### PR TITLE
[deps] Updated DRF to 3.12 #164

### LINKED ISSUE #164: [deps] Update django-rest-framework to 3.12
Before updating the dependency here, we have to test each module that depends on openwisp-utils for DRF (if any module is not depending on openwisp-utils[rest] for DRF yet, we should fix that).

- [ ] openwisp-controller
- [ ] openwisp-network-topology
- [ ] openwisp-notifications
- [ ] openwisp-monitoring
- [ ] openwisp-firmware-upgrader
- [ ] openwisp-radius (https://github.com/openwisp/openwisp-radius/issues/194)

### PR DESCRIPTION
**Pull request by ManishShah120**
I have tested all of this module's with `djangorestframework~=3.12.0`:-

- [x] openwisp-controller
- [x] openwisp-network-topology
- [x] openwisp-notifications
- [x] openwisp-monitoring
- [x] openwisp-firmware-upgrader
- [ ] openwisp-radius

except `openwisp-radius` all of them are compatible, and are passing all the tests, 
it will get fixed by https://github.com/openwisp/openwisp-radius/pull/203/

Closes #164

### CODE REVIEW

**Review by atb00ker — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -54,7 +54,7 @@
             'coveralls~=3.0.0',  # depends on coverage as well
         ],
         'rest': [
-            'djangorestframework>=3.11,<3.12',
+            'djangorestframework~=3.12.0',
```
Comment (atb00ker) [inline setup.py:57]: Important change, can you please include in changelog? :smile:

Comment (ManishShah120) [reply setup.py:57]: Done

File: CHANGES.rst [hunk 2]
```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+Version 0.7.5 [unreleased]
```
Comment (nemesisdesign) [inline CHANGES.rst:4]: change `0.7.5` to `0.8.0` please

**Review by nemesisdesign — CHANGES_REQUESTED** [review]

**Review by nemesisdesign — APPROVED** [review]
I fixed the change log to speed up. Thanks.

**Pull request merged**